tests/phpstan-polyfill.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+(function () : void {
+    static $map = [
+        'db2' => __DIR__ . '/../vendor/jetbrains/phpstorm-stubs/ibm_db2/ibm_db2.php',
+        'mysqli' => __DIR__ . '/../vendor/jetbrains/phpstorm-stubs/mysqli/mysqli.php',
+        'oci8' => __DIR__ . '/../vendor/jetbrains/phpstorm-stubs/oci8/oci8.php',
+        'sqlsrv' => __DIR__ . '/../vendor/jetbrains/phpstorm-stubs/sqlsrv/sqlsrv.php',
```

Comment:
Given that the stub files are nicely organized, in order to avoid copy/paste and simplify maintenance, can this be reworked as:
```php
$extensions = ['ibm_db2', ...];

foreach ($extensions as $extension) {
    if (extension_loaded($extension)) {
        continue;
    }

    require sprintf(__DIR__ . '/../vendor/jetbrains/phpstorm-stubs/%1$s/%1$s.php', $extension);
}
```